src/components/message-box/MessageBox.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Box, {COLOR} from '../box/Box';
+import Icon, {TYPE as iconTypes, ICON_COLOR} from '../icons/Icon';
+
+const MessageBox = ({children, onClose, ...props}) => {
+  const boxClass = classNames('sg-message-box', {
+    'sg-message-box--closable': onClose
+  });
+
+  return (
+    <Box {...props} className={boxClass} >
```

Comment:
Oh! this was good! we don't want to mix classes and overrides ! CC @maciekbb

Reply:
@maciekbb maybe Aju is right, maybe we should keep specific styles for Box in Box and here just display this specific Box, with proper padding?